app/src/main/res/values-it-rIT/strings.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="discoverButton">Scoprire</string>
+    <string name="appDescription">\"La migliore app per l\'organizzazione della casa\" -Gli sviluppatori</string>
+    <string name="settings_speakable_text">Impostazioni</string>
+    <string name="house_speakable_text">Selezione casa</string>
+    <string name="info_speakable_text">Info</string>
+    <string name="img_house_row_desc">Immagine della casa</string>
+    <string name="home_logo">Logo casa</string>
+    <string name="color_theme">Tema colore</string>
+    <string name="title_activity_settings">Impostazioni</string>
+    <string name="no_household_info">Ancora nessuna informazione!</string>
+    <string name="edit_household_notes">Modificare la descrizione</string>
+    <string name="edit_notes_button">Modificare</string>
+    <string name="info_header">Residenti</string>
+    <string name="add_household">Creare un\'abitazione</string>
+    <string name="add_join_household">Unirsi/Creare un\' abitazione</string>
+    <string name="add_household_success">Aggiunta di un\'abitazione!</string>
+    <string name="add_household_failure">Non è stato possibile aggiungere l\'abitazione</string>
+    <string name="scanQRButton">Scansiona il codice QR per unirti a la casa</string>
+    <string name="household_creation_hint">Nome dell\' abitazione</string>
+    <string name="latitude_hint">Latitudine</string>
+    <string name="longitude_hint">Longitudine</string>
+    <string name="leave_house">Partire</string>
+    <string name="house_name">Nome dell\' abitazione</string>
+    <string name="permission_denied">Questa app richiede l\'autorizzazione per poter funzionare correttamente</string>
+    <string name="changed_picture">Immagine caricata!</string>
+    <string name="change_picture_speakable_text">Cambia immagine</string>
+    <string name="user_name">Nome utente</string>
+    <string name="edit_button_description">Premere per modificare l\'abitazione</string>
+    <string name="delete_button">Cancellare l\'abitazione</string>
+    <string name="edit_title">Abitazione</string>
+    <string name="add_user">Aggiungi utente</string>
+    <string name="change_owner">Cambia proprietario</string>
+    <string name="add_user_hint">Indirizzo e-mail valido</string>
+    <string name="invalid_email">L\'e-mail fornita non è valida</string>
+    <string name="duplicate_user">Utente già presente!</string>
+    <string name="not_owner">Non sei il proprietario!</string>
+    <string name="owner_change_success">Il proprietario è stato cambiato con successo</string>
+    <string name="add_user_success">Aggiunto con successo l\'utente</string>
```

Comment:
You could say "Utente aggiunto" (line 40) but we can do language edits later & this is definitely just nitpicking

Reply:
True, my italian is rusty! I will push a fix later today, thanks for the catch. By the way @sansive if you find anything strange is spanish tell me!